types/link2aws/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for link2aws 1.0
+// Project: https://github.com/link2aws/link2aws
+// Definitions by: Kit Peters <https://github.com/popefelix>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export class ARN {
+  constructor(text: string);
+  string: string;
+  console: string;
+  qualifiers: string[];
+  pathLast: string;
+  consoleLink: string;
```

Comment:
These properties should probably use the `get` syntax to match the source code:

```suggestion
  get string(): string;
  get console(): string;
  get qualifiers(): string[];
  get pathLast(): string;
  get consoleLink(): string;
```